faster_whisper/utils.py

```diff
@@ -25,6 +25,7 @@
     "distil-large-v2": "Systran/faster-distil-whisper-large-v2",
     "distil-medium.en": "Systran/faster-distil-whisper-medium.en",
     "distil-small.en": "Systran/faster-distil-whisper-small.en",
+    "distil-large-v3": "distil-whisper/distil-large-v3-ct2",
```

Comment:
Swapping to the fp16 weights (cc @trungkienbkhn)

```suggestion
    "distil-large-v3": "Systran/faster-distil-whisper-large-v3",
```